### DISCUSSION TITLE
老版本升级4.6.8版本，咨询一下OpenAI 账号配置怎么开启，模式以前的配置不管用了。

### DISCUSSION THREAD

**Discussion by taurusduan** [labring/FastGPT#849] [Q&A]
4.4.4版本，过年了有时间想升级一下最新的版本，但是以前的OpenAI 账号配置貌似失效了。请教一下现在的版本要如何开启。

![image](https://github.com/labring/FastGPT/assets/30854760/ab725c25-a4c4-4b13-b6eb-c67f2f4948fb)

以前的配置文件
![image](https://github.com/labring/FastGPT/assets/30854760/a72de595-96df-4b34-ab29-a4d207431fc2)

现在没有这些配置了，不知道是不是被删除了。

**Comment by c121914yu** [comment]
> 4.4.4版本，过年了有时间想升级一下最新的版本，但是以前的OpenAI 账号配置貌似失效了。请教一下现在的版本要如何开启。
> 
> ![image](https://github.com/labring/FastGPT/assets/30854760/ab725c25-a4c4-4b13-b6eb-c67f2f4948fb)
> 
> 以前的配置文件
> ![image](https://github.com/labring/FastGPT/assets/30854760/a72de595-96df-4b34-ab29-a4d207431fc2)
> 
> 现在没有这些配置了，不知道是不是被删除了。
> 

已经删除
